model/Vocabulary.php

```diff
@@ -623,7 +623,8 @@ public function verifyVocabularyLanguage($lang)
      */
     public function getChangeList($prop, $clang, $offset, $limit)
     {
-      return $this->getSparql()->queryChangeList($prop, $clang, $offset, $limit);
+        $showDeprecated = $this->getConfig()->getShowDeprecatedChanges();
+        return $this->getSparql()->queryChangeList($prop, $clang, $offset, $limit, $showDeprecated);
```

Comment:
Now `getChangeList` will pass $showdeprecated on without it being able to be overriden in the signature. May be OK, but should give a thought for it before making the decision.